style/docs.css

```diff
@@ -185,3 +189,8 @@ hr {
 .warning {
   background-color: #ff9800;
 }
```

Comment:
In some places, you used variables for color, here you used hex value please use one consistent methode

Reply:
okay